### ISSUE TITLE
Retina display

### ISSUE THREAD

**Issue by gregtatum** [mattdesl/gl-vignette-background#1]
Just some thoughts, but from how I'm using it, it appears that the difference between the noise on the retina vs. non-retina is pretty striking. I liked the noise quality at non-retina, but the retina seemed to be too fine. It might be nice to scale the noise somehow or texture size.

I'm not sure if I'm just missing the option.

**Comment by mattdesl** [comment]
Yeah I've noticed that too. Because I am using glsl-random there seems to be no easy way of zooming the noise. 

I might have to switch to glsl-noise to fix. While I'm at it I will probably clean up some of the code and release 2.0.0.

**Comment by mattdesl** [comment]
`three-vignette-background` uses a different approach for noise:
https://github.com/mattdesl/three-vignette-background/blob/master/shader.frag

Both have pros/cons. The simplex noise is much more expensive, making coloured noise not really possible, but it provides more control over scale.
